cmd/deploy/remote.go

```diff
@@ -69,7 +69,8 @@ WORKDIR /okteto/src
 ARG {{ .GitCommitArgName }}
 ARG {{ .InvalidateCacheArgName }}
 
-RUN okteto deploy --log-output=json --server-name="${{ .InternalServerName }}" {{ .DeployFlags }}
+RUN ssh-keyscan {{ join .KnownSshHosts " " }} >> ~/.ssh/known_hosts
```

Comment:
@maroshii could we also apply the changes to `okteto destroy`. All the remote code is duplicated between `cmd/deploy` and `cmd/destroy`.
@jLopezbarb I think we should refactor ^^ in common interfaces, wdyt?

Reply:
Yes, we have to refactor so both codes can share as much as possible. Something that we'll look into it on next sprints